app/controllers/foreman_ansible/api/v2/hostgroups_controller_extensions.rb

```diff
@@ -50,7 +50,7 @@ def ansible_roles
             @inherited_ansible_roles = @hostgroup.inherited_ansible_roles
             @directly_assigned_roles = @hostgroup.ansible_roles
             @ansible_roles = (
-              @directly_assigned_roles + @inherited_ansible_roles + @hostgroup.host_ansible_roles
```

Comment:
That was a slight refactor of https://github.com/theforeman/foreman_ansible/blob/5b44890713842fd3c5f2646590ecb18265f52ebf/app/models/concerns/foreman_ansible/hostgroup_extensions.rb#L37-L42 usage for response. I don't think we need to change that.

Reply:
@ofedoren I believe that should be implemented as a separate API call then. I do see a strong reason behind getting a list of roles applied to a hostgroup but including roles assigned directly to hosts to that list shouldn't be default, maybe optional.

Reply:
I agree with Alexey here. 
IMHO we shouldn't return Ansible roles that are not directly assigned to the hostgroup by default.
For example, if I have a host which is a member of a hostgroup, and that host has ansible roles assigned to it, when I run `hammer hostgroup ansible-roles list --id hostgroup_id` (or via API) I will get these roles in the results, even though they are not directly assigned to that hostgroup (or to its sub-hostgroups).
I think that can be confusing.